### ISSUE TITLE
no work if youtube video is on fullscreen

### ISSUE THREAD

**Issue by ghost** [jasonfah/chrome-tab-titles#1]
if you have a youtube video on fullscreen you can't get the title.
how then to get the title?
thanks

**Comment by jasonfah** [comment]
Yeah, it looks like Chrome destroys the UI when it goes full screen. All of the UI elements are no longer present in inspect.exe which makes things difficult.

I'll do a little more digging but if the UI doesn't exist, I'm not sure it's going to be possible.

**Comment by jenokizm** [comment]
If youtube is open or something similar it is not necessary to receive tabs. It is important in the program to mark such a moment when a full-screen video is open.
